app/scripts/controllers/eventRegistrations.js

```diff
@@ -93,6 +93,32 @@ angular.module('confRegistrationWebApp')
       $scope.queryParameters.block = visibleBlocks;
     };
 
+    // Set initial built in Display Column values
+    var builtInColumnsVisible = {
```

Comment:
Can this object be published on `$scope`? You have this variable and then 6 others to hold the same state. You can use it in the angular template like `ng-if="builtInColumnsVisible.Email"`. I might even manipulate it in the template `ng-click="builtInColumnsVisible.Email = !builtInColumnsVisible.Email"`